ch04.adoc

```diff
@@ -236,25 +236,24 @@ UDUNITS permits a number of alternatives to the word **`since`** in the units of
 All the alternatives have exactly the same meaning in UDUNITS.
 For compatibility with other software, CF strongly recommends that `since` should be used.
 
-The reference datetime string (appearing after the identifier **`since`**) is required.
-It may include date alone, or date and time, or date, time and time zone offset.
-Its format is __y__-__m__-__d__ [__H__:__M__:__S__ [__Z__]], where [...] indicates an optional element,
+The reference datetime string (appearing after the identifier **`since`**) is required. It must include the date, which may optionally be followed by time or time zone offset or both. Its format is __y__-__m__-__d__ [__H__:__M__:__S__ [__Z__]], where [...] indicates an optional element:
 
 * _y_ is year, _m_ month, _d_ day, _H_ hour and _M_ minute, which are all integers of one or more digits, and _y_ may be prefixed with a sign (but note that some CF calendars do not permit negative years; see <<calendar>>),
 
 * _S_ is second, which may be integer or floating point (see <<leap-seconds>> regarding __S__>59),
 
-* _Z_ is the time zone offset with respect to UTC.
-This is an interval of time, specified in one of the formats described below.
-Only numbers (digits, `+`, `-` and `:`) are allowed in _Z_, not time zone names or acronyms.
+* _Z_ is the time zone offset. This is an interval of time, specified in one of the formats described below.
```

Comment:
New line needed (CF style, not an asciidoc rule) for the new sentence.

Reply:
Hmm - will that make it no longer part of that bullet point?

Changed -- but someone should confirm the formatting (I can't see how to do that in a PR ...)